torch/csrc/deploy/interpreter/interpreter_impl.cpp

```diff
@@ -9,6 +9,7 @@
 #include <pybind11/functional.h>
 #include <torch/csrc/DynamicTypes.h>
 #include <torch/csrc/autograd/generated/variable_factories.h>
+#include <torch/csrc/autograd/python_variable.h>
```

Comment:
this file is going away asap -- so best to not modify to avoid more merge conflicts